nc_time_axis/tests/unit/test_NetCDFTimeDateLocator.py

```diff
@@ -115,5 +115,43 @@ def test_yearly(self):
             self.check(5, 0, 5*365), [31., 485., 942., 1399., 1856.])
 
 
+class Test_tick_values_yr0(unittest.TestCase):
+    def setUp(self):
+        self.date_unit = 'days since 0001-01-01 00:00'
+
+    def check(self, max_n_ticks, num1, num2, calendar):
+        locator = NetCDFTimeDateLocator(max_n_ticks=max_n_ticks,
+                                        calendar=calendar,
+                                        date_unit=self.date_unit)
+        return locator.tick_values(num1, num2)
+
+    def test_yearly_yr0_remove(self):
+            yr0_remove_calendars = [
+                "proleptic_gregorian",
+                "gregorian",
+                "julian",
+                "standard"
+                    ]
+
+            for calendar in [
+                'standard',
+                'gregorian',
+                'proleptic_gregorian',
+                'noleap',
+                '365_day',
+                '360_day',
+                'julian',
+                'all_leap',
+                '366_day',
+                    ]:
```

Comment:
Create this list in `setUp` - it is likely that `test_yearly_yr0_remove()` will remain the only test in `Test_tick_values_yr0`, but even if we end up with other tests they will also be interested in this object.

Also valuable since we want the actual test to have the minimum possible code outside the behaviour under test.

Reply:
Done